bedrock/firefox/templates/firefox/nothing-personal/index.html

```diff
@@ -0,0 +1,149 @@
+          </div>
+          <div class="c-sticky-note c-attached-sticky">
+
+           <span aria-hidden="true">>>></span> {{ download_firefox_thanks(alt_copy='download', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='secondary cta') }} <span aria-hidden="true"> <<<< </span>
+           <br>
+           <span aria-hidden="true">>>></span> {{ download_firefox_thanks(alt_copy='download', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='secondary cta') }} <span aria-hidden="true"> <<<< </span>
+           <br>
+           <span aria-hidden="true">>>></span> {{ download_firefox_thanks(alt_copy='download', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='secondary cta') }} <span aria-hidden="true"> <<<< </span>
+            <p>Ok, bye.</p>
+
+          </div>
+        </div>
+        {% endcall %}
+    </div>
+
+    <div class="c-sticky-note c-detached-sticky">
+      <span aria-hidden="true">>>></span> {{ download_firefox_thanks(alt_copy='download', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='secondary cta') }} <span aria-hidden="true"> <<<< </span>
+      <br>
+      <span aria-hidden="true">>>></span> {{ download_firefox_thanks(alt_copy='download', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='secondary cta') }} <span aria-hidden="true"> <<<< </span>
+      <br>
+      <span aria-hidden="true">>>></span> {{ download_firefox_thanks(alt_copy='download', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='secondary cta') }} <span aria-hidden="true"> <<<< </span>
+      <p>Ok, bye.</p>
+    </div>
+
+    <div class="c-sign-off">
+      <div class="mzp-l-content c-nothing-personal">
+        <img loading="lazy" src="{{ static('img/firefox/nothing-personal/tagline.svg') }}" alt="Nothing personal. Just browsing" width="400" height="40" class="c-header-logo">
+        <div class="c-sign-off-wordmark">
+          <img loading="lazy" src="{{ static('img/firefox/nothing-personal/firefox-by-mozilla-wordmark-white.svg') }}" alt="" width="180" height="40" class="c-header-logo">
+        </div>
+      </div>
+    </div>
+    </section>
+    <aside>
+      <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ" aria-label="Link to a Youtube video" target="_blank" rel="external noopener"><img src="{{ static('img/firefox/nothing-personal/folder-icon.svg') }}" alt="">Folder <br> <.< </a>
+      <a href="https://open.spotify.com/playlist/0vvXsWCC9xrXsKd4FyS8kM" aria-label="Link to a Spotify playlist" target="_blank" rel="external noopener"><img src="{{ static('img/firefox/nothing-personal/music-icon.svg') }}" alt="">focus <br> playlist</a>
+      <a aria-hidden="true"><img src="{{ static('img/firefox/nothing-personal/bin-icon.svg') }}" alt="">MyLife</a>
```

Comment:
**issue:** If clicking this doesn't go anywhere then it doesn't need to be a link imho. It shouldn't be something I have to tab over using my keyboard. Perhaps we could just make it a `<div>` and apply a hover style to it instead?